packages/sso-express/src/controllers.ts

```diff
@@ -96,11 +96,13 @@ export const loginController =
     const codeChallenge = generators.codeChallenge(codeVerifier);
     req.session.codeVerifier = codeVerifier;
 
-    const authUrl = client.authorizationUrl({
+    let authUrl = client.authorizationUrl({
       state,
       code_challenge: codeChallenge,
       code_challenge_method: "S256",
+      ...(options.authorizationUrlParams && { ...options.authorizationUrlParams })
```

Comment:
I'd set `authrizationUrlParams: {}` in the default options in index.ts, to avoid unpacking `undefined` (even though this is valid somehow :o )